build/psgallery-release.yml

```diff
@@ -69,14 +103,16 @@ stages:
           - template: 'templates/run-pester-tests.yml'
             parameters:
               projectName: '$(Project).Tests.Unit'
+              testName: '$(Project).${{UnitTest.name}}'
 
   - stage: IntegrationTests
     displayName: Integration Tests
     dependsOn: Build
     condition: succeeded()
     jobs:
-      - job: RunIntegrationTests
-        displayName: 'Run integration tests'
+    - ${{ each IntegrationTest in parameters.IntegrationTests }}:
```

Comment:
Maybe an idea to make a job template from the `run-pester-tests.yml` so we only have to specify the loop once.

Reply:
I don't think that will work since the list of unit tests is not the same as the list of integration tests. 
For example `ApiManagement` does not have integration tests.